bang/magicollections.py

```diff
@@ -710,29 +712,26 @@ def to_fields(self, item, extra_fields=None, exclude_fields=None, order=None, *a
                 'icon': 'id',
             }))
 
-            # Add title field
-            title = item.names.get(
-                language, item.japanese_name
-                if language in settings.LANGUAGES_CANT_SPEAK_ENGLISH else item.name)
-            value = item.japanese_name
-            extra_fields.append(('card_name', {
-                'verbose_name': _('Title'),
-                'icon': 'id',
-                'type': 'title_text' if unicode(title) != unicode(value) else 'text',
-                'title': title,
-                'value': value,
-            }))
+            #Add Title
+            title = item.names.get(language, item.name if language not in settings.LANGUAGES_CANT_SPEAK_ENGLISH else None)
+            value = item.japanese_name if item.japanese_name is not None else item.name
```

Comment:
i forgot to mention, this type of check can be simplified down to

```python
item.japanese_name or item.name
```

which takes the left hand side if it's nonempty, otherwise right.